pkg/bundle/resources.go

```diff
@@ -58,6 +58,17 @@ func readResources(ctx context.Context, spec *fleet.BundleSpec, compress bool, b
 		}
 	}
 
+	// if auth doesnt have a CA cert but bundle helm Options do then
+	// lets add them here
+
+	if len(auth.CABundle) == 0 && spec.Helm.CABundle != "" {
+		cert, err := base64.StdEncoding.DecodeString(spec.Helm.CABundle)
+		if err != nil {
+			return nil, err
+		}
+		auth.CABundle = cert
+	}
```

Comment:
Weird idea (cc @StrongMonkey): what if we logged the error and continued here? Might not be a bad idea since I assume this isn't essential.